src/xdsl/printer.py

```diff
@@ -149,6 +149,12 @@ def print_attribute(self, attribute: Attribute) -> None:
             self._print(f'!i{width.data}', end='')
             return
 
+        if isinstance(attribute, FloatType):
+            width = attribute.parameters[0]
```

Comment:
As a small remark: the IRDL definitions do normally add an accessor, so you should be able to access this by name, e.g.,:

```
width = attribute.width
```